src/renderer/main.js

```diff
@@ -52,7 +52,7 @@ State.load(onState)
 // the dock icon and drag+drop.
 function onState (err, _state) {
   if (err) return onError(err)
-  state = _state
+  state = window.state = _state // Make available for easier debugging
```

Comment:
It's no longer in top-level scope because of the `require()` path change, because I moved `main.html` over to `static/`